fairlearn/postprocessing/_roc_curve_utilities.py

```diff
@@ -4,13 +4,17 @@
 import numpy as np
 import pandas as pd
 
-from ._constants import LABEL_KEY, SCORE_KEY, P0_KEY, P1_KEY
+from ._constants import (
+    LABEL_KEY, SCORE_KEY, P0_KEY, P1_KEY,
+    METRIC_DICT,
+    confusion_matrix_summary)
 from ._threshold_operation import ThresholdOperation
 
 DEGENERATE_LABELS_ERROR_MESSAGE = "Degenerate labels for sensitive feature value {}"
 
 
-def _get_roc(data, sensitive_feature_value, flip=True):
+def _get_roc(data, sensitive_feature_value, flip=False,
```

Comment:
is this very different from https://scikit-learn.org/dev/modules/generated/sklearn.metrics.roc_curve.html#sklearn.metrics.roc_curve?

Reply:
This function is in some ways more general than the sklearn's (but in some ways less general). We allow picking  `x_metric` and `y_metric` that are different from FPR and TPR, and also we allow the ability to flip the decision if that improves `y_metric`. I think we should rename this from `roc_curve` to a `tradeoff_curve`. I'll update.